Makefile

```diff
@@ -12,12 +13,15 @@ locale/downgrade.pot: downgrade
 		--from-code=utf-8 -L shell \
 		--package-name=downgrade \
 		--copyright-holder=$(AUTHOR) \
-		-o $@ $<
+		-o locale/downgrade.pot downgrade
```

Comment:
Can you keep using `$@` and `$<` instead of `locale/downgrade.pt` and `downgrade`? This way it'll keep working without any changes if we change these filenames (which is not unrealistic, given `downgrade -> downgrade.in` as a suggestion on the table)

Reply:
Yes, reverted it back.